### ISSUE TITLE
Add feedback animation for two-finger back/forward navigation.

### ISSUE THREAD

**Issue by reatlat** [wavebox/waveboxapp#805]
* Wavebox Version: [e.g. 4.2.0]
* Operating System & Version: [Mac OSX 10.14 Mojave]

**Is your feature request related to a problem? Please describe.**
no

**Is your feature request related to an account type?**
no

**Describe the solution you'd like**
Will be great to add feedback animation for page navigation on 2 fingers horizontal scroll.
maybe like Safari browser, just swipe current DOM to side or like it done on Chrome.
In Chrome Browser it has shown the dark half-circle with a white arrow.
Not sure how to explain it better, I think its only MacBook related issue :)

**Additional info**
Chrome Browser
<img width="700" alt="wavebox get cloud clever 2018-09-26 03-34-57" src="https://user-images.githubusercontent.com/9933516/46052134-50f1f480-c13d-11e8-88c8-0a436d19512b.png">

Safari Browser
<img width="650" alt="why wavebox wavebox 2018-09-26 03-36-36" src="https://user-images.githubusercontent.com/9933516/46052192-8b5b9180-c13d-11e8-9740-6c4b366a3e1f.png">

**Comment by Thomas101** [comment]
Yeah I'm with you! There's some improved mouse navigation support with the next version of electron - I'll see if I can tie it into that update

**Comment by lens0021** [comment]
I am using a third-party extension for this.
What I am using is: https://chromewebstore.google.com/detail/mkkcgajcpakpfjgikmekgnihlbhliodo
